Dockerfile

```
@@ -13,7 +13,7 @@ FROM node:24-alpine
 ENV NODE_ENV=production
 
 WORKDIR /app
-RUN apk add --no-cache ffmpeg
+RUN apk add --no-cache ffmpeg python3
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)